guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -27,6 +27,19 @@ endif::[]
 ----
 # {package-install-project} python3-pulp_manifest
 ----
++
+Note that this command stops the {Project} service and re-runs {foreman-installer}.
+Alternatively, to prevent downtime caused by stopping the service, you can use the following:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable {RepoRHEL7ServerSatelliteToolsProductVersion}
+# {foreman-maintain} packages unlock
+# yum install install python-pulp-manifest -y
```

Comment:
I am curious why the package is named differently? Above, it's called `python3-pulp_manifest`.

Reply:
The section was updated earlier and python3-pulp-manifest was introduced. This change notes that python3-pulp-manifest stops Foreman/Satellite. This change is an alternative to python3-pulp-manifest which doesn't stop Foreman/Satellite. I wasn't sure either, I was assured it was correct.

Reply:
Thanks. Your explanation would be the perfect commit message!

Reply:
@ianf77 Please, squash your commits and add your explanation above to the commit message and I'll approve this.

Reply:
Once I find out how it's done.

Reply:
@ianf77 By all means please do :)

Reply:
I have. It's done.